did-endpoint/README.md

```diff
@@ -0,0 +1,164 @@
+# did-endpoint 
+## Overview
+The `did-endpoint` is a robust, flexible, and extendable framework for managing Decentralized Identifiers (DIDs)
+
+## Purpose
+The did-endpoint aims to simplify the management of DIDs by providing a centralized service that supports multiple DID methods
```

Comment:
I think you meant descentralized service instead.

Reply:
yes thanks it was an error